perun-openapi/openapi.yml

```diff
@@ -2892,7 +2994,28 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
-#TODO getAttributesDefinitionWithRights
+  /urlinjsonout/attributesManager/getAttributesDefinitionWithRights:
+    get:
+      tags:
+        - AttributesManager
+      operationId: getAttributesDefinitionWithRights
+      summary: Returns all AttributeDefinitions for every entity and possible
+        combination of entities with rights. Only attribute definition of 
+        attributes user can read (or write) you will get.
```

Comment:
The javadoc also contains the following, please add it too:
```
Combination of entities is based on provided parameters, which are optional (at least one must be present).
```

Reply:
Summary updated.